lib/rollup.config.js

```diff
@@ -83,8 +65,9 @@ export default [
     },
     plugins: [
       resolve({ extensions: ['.js', '.jsx'] }),
-      babel(getBabelOptions({ useESModules: true })),
+      babel(getBabelOptions({ useESModules: false })),
```

Comment:
@TrySound I got a lot of warnings when bundling umd build with esModules: true. Related to external dependencies.

Reply:
Probably because you forgot to update `@babel/runtime`.

npm really suck in upgrading babel. `yarn upgrade-interactive --latest` command is simple and do not let to miss important updates.

Reply:
Also those warning messages are pretty clear IMO.